xds/internal/testutils/e2e/server.go

```diff
@@ -79,8 +88,15 @@ func StartManagementServer() (*ManagementServer, error) {
 	// Create an xDS management server and register the ADS implementation
 	// provided by it on a gRPC server. Cancelling the context passed to the
 	// server is the only way of stopping it at the end of the test.
+	serverCallbacks := v3server.CallbackFuncs{}
+	if cb != nil {
+		serverCallbacks.StreamOpenFunc = cb.StreamOpenFunc
+		serverCallbacks.StreamClosedFunc = cb.StreamClosedFunc
+		serverCallbacks.StreamRequestFunc = cb.StreamRequestFunc
+		serverCallbacks.StreamResponseFunc = cb.StreamResponseFunc
+	}
```

Comment:
Why not let the user pass a `*v3server.CallbackFuncs` instead of this intermediate type?

Reply:
I thought it would be a good idea to limit references to go-control-plane server's symbols to this package instead of having it sprinkled all over our tests.